engineer.atlas.Nyxt.yaml

```diff
@@ -0,0 +1,83 @@
+app-id: engineer.atlas.Nyxt
+# TODO Should be replaced it with org.gnome.Platform.  Shared libraries aren't
+# available from org.gnome.Platform, therefore we use the development runtime.
```

Comment:
I don't understand what the problem is here.

Reply:
this is completely broken.

.so are NEVER meant to be used outside of development, because they have versions.